apps/server/src/modules/board/service/drawing-do-authorizable.service.ts

```diff
@@ -0,0 +1,92 @@
+import { AuthorizationLoaderService } from '@modules/authorization';
+import { forwardRef, Inject, Injectable } from '@nestjs/common';
+import {
+	AnyBoardDo,
+	BoardDoAuthorizable,
+	BoardExternalReferenceType,
+	BoardRoles,
+	ColumnBoard,
+	UserWithBoardRoles,
+} from '@shared/domain/domainobject';
+import { Course } from '@shared/domain/entity';
+import { EntityId } from '@shared/domain/types';
+import { CourseRepo } from '@shared/repo';
+import { DrawingDoRepo } from '../repo/drawing-do.repo';
+
+@Injectable()
+export class DrawingDoAuthorizableService implements AuthorizationLoaderService {
```

Comment:
This authorization is not used anywhere in this module, right? Only called from file service...
I wonder if it's the correct place to have it.

The naming is wrong, as it is not for drawing element itself, but for the files.